modules/impactifyBidAdapter.js

```diff
@@ -37,9 +37,13 @@ const createOpenRtbRequest = (validBidRequests, bidderRequest) => {
     source: {tid: bidderRequest.auctionId}
   };
 
+  // Get the url parameters
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  const checkPrebid = urlParams.get('_checkPrebid');
   // Force impactify debugging parameter
-  if (window.localStorage.getItem('_im_db_bidder') != null) {
-    request.test = Number(window.localStorage.getItem('_im_db_bidder'));
```

Comment:
you are welcome to use storage.getItemFromLocalStorage, in this way, deviceAccess controls and gdpr enforcement modules will govern the access